lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -30,14 +30,16 @@ defmodule ScreenplayWeb.PaMessagesApiController do
   end
 
   def create(conn, params) do
-    with {:ok, _} <- PaMessages.create_message(params) do
+    with {:ok, message} <- PaMessages.create_message(params) do
```

Comment:
Do you think it's worth it to log failures that make it to the [FallbackController](https://github.com/mbta/screenplay/blob/main/lib/screenplay_web/controllers/fallback_controller.ex)? We may not specifically know which even failed but would be notified when they happen.